### PR TITLE
Add uitests and code cleaup for animation designer

### PR DESCRIPTION
**Pull request by ngzhian**

### CODE REVIEW

File: MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift [hunk 1]
```diff
@@ -48,6 +48,7 @@ class AnimationTableViewController: UITableViewController {
         tableView.separatorStyle = .none
         tableView.separatorColor = Config.TableViewSeparatorColor
         tableView.register(AnimationTableViewCell.self, forCellReuseIdentifier: reuseIdentifier)
+        tableView.accessibilityLabel = "Animation Table"
```
Comment (anand-sundaram) [inline MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift]: I know you are using these labels to refer to them from the tests, but are they considered magic strings?

Comment (ngzhian) [reply MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift]: yea they are, i can remove them in a next commit, working on more tests

Comment (anand-sundaram) [reply MIDIchlorians/MIDIchlorians/AnimationTableViewController.swift]: Ok. LGTM then!

**Review by anand-sundaram — APPROVED** [review]

**Comment by anand-sundaram** [comment]
LGTM!

**Comment by ngzhian** [comment]
sweet thanks for the review!

**Pull request merged**